### ISSUE TITLE
Enums not qualified in include/select

### ISSUE THREAD

**Issue by Brendonovich** [Brendonovich/prisma-client-rust#204]
Reported by @jk2kgit in discord

**Comment by FelixKLG** [comment]
I've also encoutered this issue, as per your fix in Discord I've had to specify a revision in the Cargo.toml.

**Comment by JK2Kgit** [comment]
Similar issue exists in specta type specyfications for named selects and includes introduced in #216
I think issue is here https://github.com/Brendonovich/prisma-client-rust/blob/d6fc49d72c2c442e0b084935866a8e7336940a9c/cli/src/generator/models/select.rs#L229 but i can't figure out why
[example.zip](https://github.com/Brendonovich/prisma-client-rust/files/10326196/example.zip) - smallest example i managed reproduce

And after manually expanding macro here is the error
```
200 | ...me: "EnumValue".to_string(), optional: false, ty: <TestEnum as prisma_client_rust::rspc::internal::specta::Type>::reference(prisma_cli...
    |                                                       ^^^^^^^^ not found in this scope
```

**Comment by Brendonovich** [comment]
Fixed in de2b9a96370948cf15df29c96b7c9730e7382002
